django/publicmapping/run_django.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+python manage.py migrate
+python manage.py collectstatic --noinput
```

Comment:
This command could probably be moved into the `Dockerfile` after a step that copies in the source code. Changes to the source code in a copy step would invalidate directives below it, which would allow `collectstatic` to become aware of any new static files.

See: https://github.com/azavea/collab-for-children/blob/develop/src/django/Dockerfile#L16-L19

Reply:
I have not tested this, but the static assets `collectstatic` manages should be served up in development when `DEBUG=True` if you do something like:

```python
from django.conf import settings
from django.conf.urls.static import static

urlpatterns = [
    # ... the rest of your URLconf goes here ...
] + static(settings.STATIC_URL, document_root=settings.STATIC_ROOT)
```

See: https://docs.djangoproject.com/en/1.11/howto/static-files/#serving-static-files-during-development